R/create_model_out_submit_tmpl.R

```diff
@@ -16,8 +15,11 @@
 #'
 #' @details
 #' For task IDs or output_type_ids where all values are optional, by default, columns
-#' are included as columns of `NA`s. To exclude them from the output use
-#' `remove_empty_cols = TRUE`.
+#' are included as columns of `NA`s when `required_vals_only = TRUE`.
+#' When such columns exist, the function returns a tibble with zero rows, as no
+#' complete cases of combination of required values exists. To return a template of
+#' incomplete required cases, which includes `NA` columns, use ``
```

Comment:
There is some missing documentation here.

Reply:
Doh! Fixed in f806a12e2deb27e69603b8a6ec677379d7f28d7c

One thought I just had is that, if a hub is set up with `NA` as a required and valid value for a task ID, this will cause false negatives when determining complete cases.

Do you think that might be be a reasonable set up for a hub that we need to have guardrails for? (There is a pretty simple fix I can think of for it)

Reply:
I cannot think of a scenario where we would have `NA` as a required value for a task ID. I almost think we should have a check that prevents that from being in place?